inputs/metadata-parse-paired-mgx-mtx.ipynb

```diff
@@ -0,0 +1,5079 @@
+{
```

Comment:
<p>Can we combine <code>human_gut</code> and <code>human_gut_microbiome</code> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Arcadia-Science/2022-mtx-not-in-mgx-pairs/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='97'/>

Reply:
<p>yes great catch, thank you!</p>

Reply:
I also removed `_microbiome` from `snail_gut` bc they're all microbiomes and that was the only one that still had it in it.